shell/edit/workload/index.vue

```diff
@@ -901,8 +901,8 @@ export default {
         :mode="mode"
         @change="name=value.metadata.name"
       />
-      <div class="row mb-10">
-        <div v-if="isCronJob" class="col span-4">
+      <div v-if="isCronJob || isReplicable || (isReplicable && isStatefulSet) || containerOptions.length > 1" class="row mb-20">
```

Comment:
Why are we checking if the workload is both replicable and a StatefulSet? A StatefulSet would always be replicable.

Reply:
This matches the field below